### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by Flameancer** [johnrickman/LiquidCrystal_I2C#77]
Just getting into ESP32 and I2C boards and one of the guides I'm using uses this library. However I am looking for documentation on the library and the gitlab link this repo links to is dead and leaves a 404. 

Is this still an active project and if so is there a link to documentation?

**Comment by SrGesus** [comment]
Yep, look at https://github.com/johnrickman/LiquidCrystal_I2C/issues/47#issuecomment-748690377

**Comment by SrGesus** [comment]
The least dead one i seem to find is https://github.com/enjoyneering/LiquidCrystal_I2C but the last commit was a year ago (as opposed to 4 years in the other ones)
